README.md

```diff
@@ -266,6 +266,15 @@ If you see SQL errors after upgrading please remember to check for this specific
 
 UPS shut down their old CGI APIs so we removed the support for it from the Mage_Usa module.
 
+### Between OpenMage 20.x and 21.x (unreleased, available on branch `next`)
+
+- PHP 8.1 as minimum required version
+- Removed scriptaculous/dragdrop.js (#3215)
+- RWD theme: updated jQuery to 3.7.0 (#3204)
+- Unified CSRF configuration (#3147) and added form key validation to Contacts form (#3146)
+- Removed double span element from HTML buttons (#3123)
+- Removed all deprecated Mysql4_ classes (#2730). If you have old modules/extensions that still use such classes, you have to run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script.
```

Comment:
"If there are any old modules/extensions in your installation that use such classes, you must run 'shell/rename-mysql4-class-to-resource.php` in the command line in order to convert them. Backup all files before running the script"

The wording sounds better. I removed the dot at the end of the bullet in order to keep the same format used in the readme file.

OFFTOPIC: the script name it is too long compared to the other existing ones. rename-mysql4 or mysql4 were more than enough.

Reply:
The use of the word "rename" in the name of the file is not suitable for the action that the script does. In addition, the name must be as short as possible. If we look in the /shell directory we see how simple the file names are (indexer.php, log.php). I think the most suitable would be mysql4.php.

Reply:
mmm no i don't agree, mysql4.php really would be impossible to understand, also, this is a one shot usage file and it's pontentially dangerous, the filename should be at least understandable